ios/FluentUI.Demo/FluentUI.Demo/Demos/NotificationViewDemoController.swift

```diff
@@ -248,11 +247,66 @@ class NotificationViewDemoController: DemoController {
             preconditionFailure("showNotificationView is used for a button in the wrong container")
         }
 
-        createNotificationView(forVariant: variant).show(in: view) { $0.hide(after: 3.0) }
+        let notification = createNotificationView(forVariant: variant)
+        if variant == .warningToastWithFlexibleWidth {
+            showNotification(notification: notification)
+        } else {
+            notification.show(in: view) { $0.hide(after: 3.0) }
+        }
     }
 
     @objc private func showSwiftUIDemo() {
         navigationController?.pushViewController(NotificationViewDemoControllerSwiftUI(),
                                                  animated: true)
     }
+
+    private func showNotification(notification: MSFNotification) {
+        view.addSubview(notification)
+
+        notification.translatesAutoresizingMaskIntoConstraints = false
+        let presentationOffset = notification.tokenSet[.presentationOffset].float
+        let safeAreaBottom = view.safeAreaLayoutGuide.bottomAnchor
+        let constraintWhenHidden = notification.topAnchor.constraint(equalTo: safeAreaBottom)
+        let constraintWhenShown = notification.bottomAnchor.constraint(equalTo: safeAreaBottom, constant: -presentationOffset)
+
+        NSLayoutConstraint.activate([
+            constraintWhenHidden,
+            notification.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            notification.widthAnchor.constraint(equalTo: view.widthAnchor, constant: -2 * presentationOffset )
+        ])
+        notification.alpha = 0
+        view.layoutIfNeeded()
+        flexibleWidthNotificationHidden = constraintWhenHidden
+        flexibleWidthNotificationShown = constraintWhenShown
+        UIView.animate(withDuration: 0.6,
```

Comment:
Should these values be a constant somewhere? and with the other values around the PR